### PR TITLE
Add "Listen Now" button to YouTube Music Presence 

### PR DESCRIPTION
**Pull request by Atharv-Attri**
This PR seeks to add a "Listen Now" Button in the Youtube Music presence. 

Before:
image

After:
image

The button will take you the song that the person is currently listening to. 

This PR was co-authored by @plskz.

### CODE REVIEW

File: websites/Y/YouTube Music/dist/metadata.json [hunk 1]
```diff
@@ -3,6 +3,16 @@
...
+  "contributors": [
+    {
+      "name": "Ai",
+      "id": "90431685472038912"
+    },
+    {
+      "name": "Atharv Attri",
+      "id": "679100560720396339"
+    }
+  ],
```
Comment (Bas950) [inline websites/Y/YouTube Music/dist/metadata.json:15]: this is not enough to add yourself as contributor, and not your friend either..

File: websites/Y/YouTube Music/presence.ts [hunk 2]
```diff
@@ -114,7 +114,13 @@ presence.on("UpdateData", async () => {
           ? "Playlist on loop"
           : (await strings).play,
         startTimestamp: timestamps[0],
-        endTimestamp: timestamps[1]
+        endTimestamp: timestamps[1],
+        buttons: [
+          {
+            label: "Listen now",
```
Comment (Bas950) [inline websites/Y/YouTube Music/presence.ts:120]:
```suggestion
            label: "Listen along!",
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
bump version

**Comment by Atharv-Attri** [comment]
What should I bump the version to? I'm not entirely sure how big of a change this would count as

**Comment by Bas950** [comment]
> What should I bump the version to? I'm not entirely sure how big of a change this would count as

Just bump the last number once

**Comment by Atharv-Attri** [comment]
Done!

**Review by Bas950 — DISMISSED** [review]

File: websites/Y/YouTube Music/presence.ts [hunk 3]
```diff
@@ -114,7 +114,13 @@ presence.on("UpdateData", async () => {
           ? "Playlist on loop"
           : (await strings).play,
         startTimestamp: timestamps[0],
-        endTimestamp: timestamps[1]
+        endTimestamp: timestamps[1],
+        buttons: [
+          {
+            label: "Listen along!",
```
Comment (ririxidev) [inline websites/Y/YouTube Music/presence.ts:120]:
```suggestion
            label: "Listen Along",
```
looks better :////

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**